yarn-project/pxe/src/database/outgoing_note_dao.test.ts

```diff
@@ -9,9 +9,22 @@ export const randomOutgoingNoteDao = ({
   txHash = randomTxHash(),
   storageSlot = Fr.random(),
   noteTypeId = Fr.random(),
+  nonce = Fr.random(),
+  innerNoteHash = Fr.random(),
+  index = Fr.random().toBigInt(),
   ovpkM = Point.random(),
 }: Partial<OutgoingNoteDao> = {}) => {
-  return new OutgoingNoteDao(note, contractAddress, storageSlot, noteTypeId, txHash, ovpkM);
+  return new OutgoingNoteDao(
```

Comment:
OutgoingNoteDao is the same as Incoming one other than the missing `siloedNullifier`.